examples/build/main.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	v1 "github.com/openshift/api/build/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	buildv1 "github.com/openshift/client-go/build/clientset/versioned/typed/build/v1"
+	"path/filepath"
```

Comment:
standard imports go first and then a newline, then kube, then a newline, then openshift